third_party/css/github-markdown.css

```
@@ -393,6 +393,11 @@
   /* TODO: Investigate if this should be fixed in dartdoc's styles.css: the `pre` rule has precendece over the dark theme there. */
   background-color: inherit;
 }
+.dark-theme .hljs .hljs-deletion,
```

newline above this line?
```suggestion

.dark-theme .hljs .hljs-deletion,
```